build.sbt

```diff
@@ -403,7 +403,7 @@ lazy val examples = project
   .settings(commonSettings)
   .settings(
     publish / skip     := true,
-    run / fork         := true,
+//    run / fork         := true,
```

Comment:
Restore that? Or is there a reason to remove it?

Reply:
Oh I needed it for debugging purposes since intellij won't run the debugger properly when it is forked